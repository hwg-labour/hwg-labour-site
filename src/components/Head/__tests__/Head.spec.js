import Head from "../index";
import React from "react";
import { shallow } from "enzyme";

const EVENTS_MOCK = {
  edges: [
    {
      node: {
        event: {
          Date
        }
      }
    }
  ]
};

describe("Head", () => {
  it("renders component", () => {
    const wrapper = shallow(<Head events={EVENTS_MOCK} />);

    expect(wrapper).toMatchSnapshot();
  });
});
