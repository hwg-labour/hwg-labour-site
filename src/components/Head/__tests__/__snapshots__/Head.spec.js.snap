// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders component 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
  meta={
    Array [
      Object {
        "content": "Hornsey and Wood Green Labour Party Website",
        "name": "description",
      },
      Object {
        "content": "Hornsey, Wood green, Labour, Party, Crouch End, Highgate, Alexandra, Stroud Green",
        "name": "keywords",
      },
      Object {
        "content": "Zg3zItwnIwzK5-NXu7KC5JuBu6PM4CY_nkxU78-CQtQ",
        "name": "google-site-verification",
      },
    ]
  }
  title="Hornsey & Wood Green Labour"
/>
`;
